src/layers/core/point-density-hexagon-layer/point-density-hexagon-layer.js

```diff
@@ -110,15 +150,26 @@ export default class PointDensityHexagonLayer extends Layer {
 
   _onGetSublayerColor(cell) {
     const {colorRange} = this.props;
-    const colorDomain = this.props.colorDomain || this.state.countRange;
+    const {countDomainByPercentile} = this.state;
+    const colorDomain = this.props.colorDomain || countDomainByPercentile;
+    const count = cell.points.length;
+
+    const color = ordinalScale(colorDomain, colorRange, count);
```

Comment:
Maybe some comment about what `ordinalScale` does and why it is used here. Even if we know our D3, always good to leave a hint for the next person to read this.